<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScreenshotDialog</class>
 <widget class="QDialog" name="ScreenshotDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>768</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Screenshot</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>4</number>
   </property>
   <item row="1" column="0">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>396</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Open with</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QToolButton" name="openWithButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="popupMode">
      <enum>QToolButton::MenuButtonPopup</enum>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextBesideIcon</enum>
     </property>
     <property name="arrowType">
      <enum>Qt::NoArrow</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="copyButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Copy the image to clipboard</string>
     </property>
     <property name="text">
      <string>Copy</string>
     </property>
     <property name="icon">
      <iconset theme="edit-copy">
       <normaloff/>
      </iconset>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>396</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Send to</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QToolButton" name="sendToButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="popupMode">
      <enum>QToolButton::MenuButtonPopup</enum>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextBesideIcon</enum>
     </property>
     <property name="arrowType">
      <enum>Qt::NoArrow</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QPushButton" name="saveAsButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
     <property name="icon">
      <iconset theme="document-save">
       <normaloff/>
      </iconset>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QPushButton" name="closeButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Close</string>
     </property>
     <property name="icon">
      <iconset theme="window-close">
       <normaloff/>
      </iconset>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="5">
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>758</width>
        <height>432</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="imageLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>200</height>
          </size>
         </property>
         <property name="text">
          <string>Image</string>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>saveAsButton</tabstop>
  <tabstop>closeButton</tabstop>
  <tabstop>openWithButton</tabstop>
  <tabstop>sendToButton</tabstop>
  <tabstop>copyButton</tabstop>
  <tabstop>scrollArea</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>ScreenshotDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>725</x>
     <y>496</y>
    </hint>
    <hint type="destinationlabel">
     <x>383</x>
     <y>259</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
